src/status_im/ui/screens/popover/views.cljs

```diff
@@ -95,8 +96,10 @@
       :reagent-render
       (fn []
         (when @current-popover
-          (let [{:keys [view style]} @current-popover]
-            [react/view {:position :absolute :top 0 :bottom 0 :left 0 :right 0}
+          (let [{:keys [view style disable-touchable-overlay? blur-view? blur-view-props]} @current-popover
+                comp (if blur-view? react/blur-view react/view)
```

Comment:
consider using `component` rather than `comp` here. Best to be explicit with naming imo :)

Reply:
Changed!